Rules.md

```diff
@@ -5,6 +5,7 @@
 * [Array Init](#array-init)
 * [Attributes](#attributes)
 * [Block Based KVO](#block-based-kvo)
+* [Call Super Only](#call-super-only)
```

Comment:
What about naming the rule `empty_method_override` similar to the already existing empty prefixed rules? Since the call to super doesn't actually do anything, the method can be considered empty.

Alternatively, `unused_method_override`, `unused_method_declaration` or `unused_method_body` might also work. I don't like th ename `call_super_only` because it reads like an invitation to call super and might be confused to be a rule, that ensures you don't forget to call super in methods decared with `override`.

Reply:
Yes, these are great suggestions, thank you! 👍

I'm going to go for `empty_method_override`.